ecospheres_migrator/geonetwork.py

```diff
@@ -99,7 +99,7 @@ def get_records(self, query=None) -> list[Record]:
             "_content_type": "json",
             "buildSummary": "false",
             "fast": "index",  # needed to get info such as title
-            "sortBy": "title",  # FIXME: or changeDate?
+            "sortBy": "changeDate",
             "sortOrder": "reverse",
```

Comment:
"reverse" is what we want for processing, but not for preview. For preview I'd expect to see the same list as what I see in the "editor board" on GN, which lists most recent first by default.
IMO we should have standard sortOrder by default, and either re-query with "reverse" for processing, or (prob. better) internally reverse our list before processing. It gets tricky to display processing results though (prepend in results table?)...

Reply:
I see your point, but do we really want to put that much effort into this?

If we do and need multiple ways to sort the list, I would probably extract the `changeDate` from the record and store it in the batch records, so that we can sort explicitly on it without reprocessing the XML. There would still be some hair-splitting to do when dealing with failed and skipped migration record (we would probably need the original `changeDate` in this case, vs the post-migration `changeDate` for successes).

Reply:
I find it unsettling but not sure how much effort it's worth...
How about adding a small comment before the tables (= preview and results) saying something like "Les enregistrements sont listés dans l'ordre de traitement, du plus ancien au plus récent". And we'll wait for feedback to decide whether we need to do more.